featurebyte/query_graph/sql/parent_serving.py

```diff
@@ -88,7 +102,9 @@ def construct_request_table_with_parent_entities(
     return ParentEntityLookupResult(
         table_expr=table_expr,
         parent_entity_columns=new_columns,
-        new_request_table_name="JOINED_PARENTS_" + request_table_name,
+        new_request_table_name=(
+            "JOINED_PARENTS_" + request_table_name if request_table_name else "JOINED_PARENTS"
+        ),
```

Comment:
Should we include relationship hash in the table name?

Reply:
We don't need to. As long as this is a table name that is different from the original request table name, it should be sufficient.